packages/anoncreds/tests/fixtures/presentation-definition.ts

```diff
@@ -0,0 +1,56 @@
+import type { PresentationDefinitionV1 } from '@sphereon/pex-models'
```

Comment:
Same here, would be good to use as much types from the `dif-presentation-exchange` module in core, to prevent dependency on the sphereon libs all across the codebase (same like we try to hide AnonCreds RS implementation behind AnonCreds generic interface)